datastruct/elasticarray.c

```diff
@@ -255,6 +255,20 @@ elasticarray_get(struct elasticarray * EA, size_t pos, size_t reclen)
 	return ((uint8_t *)(EA->buf) + pos * reclen);
 }
 
+/**
+ * elasticarray_iter(EA, reclen, fp):
+ * Run the ${fp} function on every member of the array.
+ */
+void
+elasticarray_iter(struct elasticarray * EA, size_t reclen, void(fp)(void *))
```

Comment:
I think I normally spell the last parameter `void (* fp)(void *)`.  This might be a case like arrays where the C language infers a pointer even when not explicit... but let's be consistent anyway.